### PR TITLE
Add Firehydrant docs

### PR DESCRIPTION
**Pull request by Xantier**
App: https://github.com/RoadieHQ/roadie/pull/228/files
Docs: https://github.com/RoadieHQ/marketing-site/pull/479/files
Config: https://github.com/RoadieHQ/roadie-charts/pull/126

### CODE REVIEW

File: content/docs/integrations/firehydrant/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+https://<tenant-name>.roadie.so/administration/settings/secrets
+```
+
+This page describes how to create and set up the FireHydrant Personal Access (API) token.
+
+## Steps
+
+### Step 1: Create an API token
+
+Start by visiting the account settings page in FireHydrant :https://app.firehydrant.io/organizations/bots.
```
Comment (dtuite) [inline content/docs/integrations/firehydrant/index.md:24]:
```suggestion
Start by visiting the [account settings page in FireHydrant](https://app.firehydrant.io/organizations/bots).
```

File: content/docs/integrations/firehydrant/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+After you have added your preferred name and description to the bot, you should be presented with a bot token. Copy this token to clipboard and continue to step 2
+
+### Step 2: Store the token in Roadie
+Visit `https://<tenant-name>.roadie.so/administration/settings/secrets` and enter the token value from above into `FIREHYDRANT_BOT_TOKEN`.
+
+### Step 3: Configure FireHydrant services to match Roadie service names
+
+FireHydrant plugin uses the fully qualified entity name to identify a service in FireHydrant. This consists of kind, name, and namespace. An example of this would be `Component:default/sample-service-5` which would be pointing to https://github.com/RoadieHQ/sample-service/blob/main/catalog-info-5.yaml.
+
+You can create services in FireHydrant by navigating to URL https://app.firehydrant.io/services/new
```
Comment (dtuite) [inline content/docs/integrations/firehydrant/index.md:39]:
```suggestion
You can create services in FireHydrant on the [New Service page](https://app.firehydrant.io/services/new).
```

File: content/docs/integrations/firehydrant/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+   ![Bot Tokens screen in FireHydrant with no information added](./firehydrant_token.png)
+
+After you have added your preferred name and description to the bot, you should be presented with a bot token. Copy this token to clipboard and continue to step 2
+
+### Step 2: Store the token in Roadie
+Visit `https://<tenant-name>.roadie.so/administration/settings/secrets` and enter the token value from above into `FIREHYDRANT_BOT_TOKEN`.
+
+### Step 3: Configure FireHydrant services to match Roadie service names
+
+FireHydrant plugin uses the fully qualified entity name to identify a service in FireHydrant. This consists of kind, name, and namespace. An example of this would be `Component:default/sample-service-5` which would be pointing to https://github.com/RoadieHQ/sample-service/blob/main/catalog-info-5.yaml.
```
Comment (dtuite) [inline content/docs/integrations/firehydrant/index.md:37]: > which would be pointing to https://github.com/RoadieHQ/sample-service/blob/main/catalog-info-5.yaml

I'm struggling to understand this. Perhaps we could add an example annotation?

**Review by dtuite — APPROVED** [review]
I'm approving for expediency and we can open a separate PR for fixes.

This needs a sidebar link in the docs page.

File: content/docs/integrations/firehydrant/index.md (hunk 3 above)
Comment (Xantier) [reply content/docs/integrations/firehydrant/index.md]: there's no annotation for this plugin, it just uses the actual entity triplet of a backstage item. I modified the text a little bit.

**Review by dtuite — APPROVED** [review]

**Pull request merged**